metricbeat/metricbeat.reference.yml

```diff
@@ -251,7 +253,8 @@ metricbeat.modules:
 
 #-------------------------------- HTTP Module --------------------------------
 - module: http
-  metricsets: ["json"]
+  #metricsets:
```

Comment:
Should we in the reference file have the `enabled` one commented out to make it more clear? I like it in the short config to make sure the defaults keep working but here it's more about documenting it?